web/react/components/sidebar.jsx

```diff
@@ -49,8 +52,44 @@ export default class Sidebar extends React.Component {
         state.loadingDMChannel = -1;
         state.windowWidth = Utils.windowWidth();
 
+        this.unreadCount = this.setUnreadCount();
         this.state = state;
     }
+    setUnreadCount() {
```

Comment:
This is called `setUnreadCount` but it's not actually setting anything, it's really just getting the unread counts. Maybe make this function do the actual setting too instead of returning the counts?